pylas/headers/rawheader.py

```diff
@@ -1,9 +1,11 @@
 import ctypes
 import datetime
-import warnings
+import logging
 
 from .. import errors
 
+logger = logging.getLogger(__name__)
```

Comment:
This isn't exactly what you've done in 'pylas.lib', but I've always preferred seeing the `__name__` when reviewing logs.

Reply:
That's nice, I tried doing the same thing in pylas.lib but it changes the format of the ouput message, I don't really understand why 🤔